JWT/JWT.cs

```diff
@@ -173,6 +173,25 @@ private static void Verify(string decodedCrypto, string decodedSignature, string
                     throw new SignatureVerificationException("Token has expired.");
                 }
             }
+            // verify nbf claim https://tools.ietf.org/html/draft-ietf-oauth-json-web-token-32#section-4.1.5
+            if (payloadData.ContainsKey("nbf") && payloadData["nbf"] != null)
```

Comment:
hi, do you still care about this pr? Would you mind to review/refactor it once again?